tests/phpunit/integration/api/actions/beansAddAction.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * Tests for beans_add_action()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Actions
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Actions;
+
+use WP_UnitTestCase;
+
+/**
+ * Class Tests_BeansAddAction
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
+ * @group   unit-integration
+ * @group   api
+ */
+class Tests_BeansAddAction extends WP_UnitTestCase {
```

Comment:
Ignore this function.  It's part of PR #92.  It's shown here as I built this PR off of PR #92's branch.